pkg/network/node/node.go

```diff
@@ -366,6 +401,19 @@ func (node *OsdnNode) Start() error {
 	}
 
 	klog.V(2).Infof("openshift-sdn network plugin ready")
+
+	klog.V(2).Infof("Checking default interface MTU")
+	mtu, err := getDefaultMTU()
+
+	if err != nil {
+		return err
+	}
+
+	if node.mtu < mtu+50 {
+		klog.V(2).Infof("Configured node MTU is less than default interface MTU plus VXLAN overhead, tainting node...")
```

Comment:
This is backwards; `node.mtu` is the MTU for the VXLAN tunnel. It must be *smaller* than the default interface MTU.